integtest/spec/all_books_change_detection_spec.rb

```diff
@@ -0,0 +1,275 @@
+# frozen_string_literal: true
+
+##
+# Assertions about when books are rebuilt based on changes in source
+# repositories or the book's configuration.
+RSpec.describe 'building all books' do
+  describe 'change detection' do
```

Comment:
I really just renamed the file, added this line and indented everyone once.

Reply:
yeah, that works!